IPs/RSKIP428.md

```diff
@@ -0,0 +1,96 @@
+---
+rskip: 428
+title: Emit a new event with the utxo outpoints when a peg-out is created
+created: 23-APR-24
+author: NC
+purpose: Sca
+layer: Core 
+complexity: 2
+status: Draft
+description: 
+---
+
+|RSKIP          | 428                |
+| :------------ |:-------------------|
+|**Title**      | Emit a new event with the utxo outpoints when a peg-out is created |
+|**Created**    | 23-APR-24          |
+|**Author**     | NC                 |
+|**Purpose**    | Sca                |
+|**Layer**      | Core               |
+|**Complexity** | 2                  |
+|**Status**     | Draft              |
+
+## Abstract
+
+This RSKIP proposes creating a new event that stores the outpoint values of the inputs used in a 
+peg-out to enable the PowHSM sign segwit peg-outs.
+
+## Motivation
```

Comment:
Not sure, but it might be worth it to include a brief explanation on why the outpoint values are needed for the PowHSM to sign segwit pegouts

Reply:
Done